pkg/connectorrunner/runner.go

```diff
@@ -194,6 +197,14 @@ func (c *connectorRunner) run(ctx context.Context) error {
 				defer sem.Release(1)
 				err := c.processTask(ctx, t)
 				if err != nil {
+					switch code := status.Code(err); code {
+					case codes.Canceled:
+						commErrCount++
+					case codes.Unknown:
```

Comment:
We wrap some http errors in grpc status Unknown: https://github.com/ConductorOne/baton-sdk/blob/d8b900d/pkg/uhttp/wrapper.go#L294

Maybe it makes sense to only increment the error count if it's a grpc client connection error.